osaca/osaca.py

```diff
@@ -65,71 +65,70 @@ def include_ibench(self):
         file.
         """
         # Check args and exit program if something's wrong
-        if(not self.check_arch()):
+        if not self.check_arch():
             print('Invalid microarchitecture.', file=sys.stderr)
             sys.exit(1)
-        if(not self.check_file()):
+        if not self.check_file():
             print('Invalid file path or file format.', file=sys.stderr)
             sys.exit(1)
         # Check for database for the chosen architecture
         self.df = self.read_csv()
-        # Create sequence of numbers and their reciprokals for validate the measurements
+        # Create sequence of numbers and their reciprocals for validate the measurements
         cyc_list, reci_list = self.create_sequences()
         print('Everything seems fine! Let\'s start!', file=self.file_output)
         new_data = []
         added_vals = 0
         for line in self.srcCode:
-            if('Using frequency' in line or len(line) == 0):
+            if 'Using frequency' in line or len(line) == 0:
                 continue
             clmn = 'LT'
```

Comment:
column? Please use (mostly) complete words, spaced with underscores.

Reply:
changed `clmn` to `column`